api/Window.json

```diff
@@ -593,6 +593,54 @@
           }
         }
       },
+      "beforeinstallprompt_event": {
+        "__compat": {
+          "description": "<code>beforeinstallprompt</code> event",
+          "support": {
+            "chrome": {
+              "version_added": "45"
```

Comment:
Given https://github.com/mdn/browser-compat-data/issues/7844 I especially distrust Chrome 45 values so I dug deeper.

The feature was enabled by https://chromium.googlesource.com/chromium/src/+/e9e94d1355522e4e4a5a8aa2f540d265130ab98c which via https://www.chromium.org/developers/calendar maps to Chrome 44, although it's so close to the branch date it could have been 45.

But I've also tested logging `BeforeInstallPromptEvent` in BrowserStack, and can see that it was introduced in Chrome 44.

Can you update the data for `BeforeInstallPromptEvent` to 44 and also update this to match?